lighthouse-plugin-publisher-ads/audits/ad-request-critical-path.js

```diff
@@ -43,16 +43,16 @@ const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
  * @type {LH.Audit.Details.Table['headings']}
  */
 const HEADINGS = [
+  {
+    key: 'nameOrTld',
```

Comment:
Maybe better just as "name" here? This is part of the JSON report output and without context I think this would be confusing for a user.

Reply:
I think that's fine, we don't support that and change it without warning. If anything, this is more descriptive of what goes in here